### PR TITLE
docs(update): updated the docs of API.md with reference to API.js test file

### PR DESCRIPTION
**Pull request by Cronus1007**
## Issue that this pull request solves

Closes: #403 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.
@bbangert As per our discussion related to the issue #403 I have updated the documentation and soon going to come with information regarding the updation of the documentation automatically.

### CODE REVIEW

File: packages/fxa-profile-server/docs/API.md [hunk 1]
```diff
@@ -68,6 +68,10 @@ The currently-defined error responses are:
 - [GET /v1/ecosystem_anon_id][ecosystem_anon_id]
 - [POST /v1/ecosystem_anon_id][ecosystem_anon_id-post]
 - [DELETE /v1/cache/:uid][cache_delete]
+- [GET /v1/\_core_profile][hidden from extra callers by default]
```
Comment (Cronus1007) [inline packages/fxa-profile-server/docs/API.md:71]: @bbangert  Whenever I am saving this file this additional \ occurs due to no reason and even if we save without formatting this problem is going to arise due to `husky pre commit `

File: packages/fxa-profile-server/docs/API.md [hunk 2]
```diff
@@ -391,3 +395,83 @@ Delete cached profile data for user `:uid`
 [ecosystem_anon_id-post]: #post-v1ecosystem_anon_id
 [oauth]: https://github.com/mozilla/fxa-oauth-server
 [cache_delete]: #delete-v1cache:uid
+
+### GET /v1/\_core_profile
```
Comment (Cronus1007) [inline packages/fxa-profile-server/docs/API.md:399]: @bbangert  @chenba  Addded the important information related to the enhancement of API.md in the fxa-profile-server

**Comment by Cronus1007** [comment]
cc @chenba  @bbangert  @dannycoates  @meandavejustice

**Comment by 6a68** [comment]
Hi @Cronus1007 - Thanks for opening this pull request! I'm sorry it's been sitting without feedback for such a long time.

After looking at #403 a bit, it seems like an unfairly difficult issue for you to work on. Switching FxA to use OpenAPI schemas, then generating docs from those schemas, would be a large, complex, and ill-defined project for a full-time engineer, and all but impossible for a community contributor.

I appreciate your persistence in working on this issue, and in trying to get our collective attention; sorry again that it's taken so long to get back to you.

Here's what I'd suggest: have a look at the issues labeled good-first-issue here: https://github.com/mozilla/fxa/labels/good%20first%20issue and see if any of them seem interesting. Those are all important, and more clearly defined contributions you can make, and they should be a much more manageable size. If you have questions, or would like to work on one of them, you can either ask me in the bug, or drop by our channel on Matrix (https://chat.mozilla.org/#/room/#fxa:mozilla.org) to ask questions there. I'm on Pacific time (GMT-8), so there may not be an immediate response, depending on your timezone, but we can still chat async. :-)

I'm going to close this PR for now, but I hope to see you tackle another FxA issue soon!

**Pull request closed without merging**